flexibleengine/resource_flexibleengine_s3_bucket.go

```diff
@@ -140,23 +139,21 @@ func resourceS3Bucket() *schema.Resource {
 
 			"hosted_zone_id": {
 				Type:     schema.TypeString,
-				Optional: true,
 				Computed: true,
 			},
 
 			"region": {
 				Type:     schema.TypeString,
 				Optional: true,
+				ForceNew: true,
 				Computed: true,
 			},
 			"website_endpoint": {
 				Type:     schema.TypeString,
-				Optional: true,
```

Comment:
it's better to put all `computed` attrs at the end of schema.

Reply:
I have fix the problem, thanks for your advice